### DISCUSSION TITLE
How to make this work with IDE extensions?

### DISCUSSION THREAD

**Discussion by AlperShal** [pelme/htpy#20] [Q&A] [answered]
Hi! I REALLY liked this project the moment I have seen it. But! The thing is it doesn't play along with the already available extensions for example for VS Code. I at least need Tailwind CSS IntelliSense, Tailwind Fold and htmx-tags extensions to develop for the last project I have been working on so it is more efficient. Is it impossible if we do not develop extensions specifically for HtPy or update the currently available ones?

**Comment by pelme** [comment, marked as answer]
Hi @AlperShal, I am glad you find the project useful! :) 

It would be great to have editor support for these but it is not something I would personally be able to work on for now.

If you feel like experimenting with this, please post your findings here. If you make a plugin or contribute to existing plugins we can add it to the docs to make it easier to coordinate/help others find it.

**Reply by AlperShal** [reply]
Yeah sure. I am now working for my university entrance exam so it won't be really possible for me to work on this for the next 40 days. After that I may. Will do post my findings if I do so.

**Comment by pelme** [comment]
Btw, how did you find out about htpy?

**Reply by AlperShal** [reply]
That's a nice question but I really don't remember the exact source even though it's been just two days. :) It was from a webpage listing alternatives to Jinja2. Which I don't think I have searched in a search engine since I can not find any result in my search history. Probably some comment from Reddit made me open that page.
